laravel/jetstream
# Dark Mode v2

This PR brings back Dark Mode changes introduced in [PR 1209](https://github.com/laravel/jetstream/pull/1209) without causing issues to existing applications with Jetstream installed as it was mentioned in [Issue 1226](https://github.com/laravel/jetstream/issues/1226).

## Changes
- As suggested by @jessarcher in [this comment](https://github.com/laravel/jetstream/issues/1226#issuecomment-1396196078), the views from `views/resources` remain unchanged to avoid issues with previous installs.

-  Original files from `views/resources` have been copied to `stubs/livewire/resources/views/` to keep track of changes.

- On new installations, the following directories `stubs/livewire/resources/views/components` `stubs/livewire/resources/views/mail` will be copied to `views/vendor/jetstream/`.

- On new installations without `--dark` directive, the command `php artisan vendor:publish --tag=jetstream-views --force` was removed. Views are already published as mentioned in point above.

- The changes introduced to strip dark classes [PR 1224](https://github.com/laravel/jetstream/pull/1224), dark nav menu in [PR 1225](https://github.com/laravel/jetstream/pull/1225) have been included.

## Step to verify Issue 1226 is Fixed

1. Install new laravel app
2. Install Jetstream 2.16 `composer require laravel/jetstream`
3. Run ```php artisan jetstream:install livewire --teams```
4. Check that there are not any published jetstream views in `views/vendor/jetstream`
5. Update Jetstream to `"laravel/jetstream": "dev-dark-mode-v2"` 
6. Include the PR repo.
```
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/fotrino/jetstream.git"
        }
      ],
```
7. Run `composer update`
